data/mods/blazemod/blaze_blob.json

```diff
@@ -431,20 +431,29 @@
   {
     "id": "gloople_tank",
     "type": "CONTAINER",
-    "name": "gelatinous tank",
-    "name_plural": "gelatinous tanks",
+    "name": "gelatinous capsule",
+    "name_plural": "gelatinous capsules",
     "description": "While the blob is very eager to be fed, it's not as enthusiastic about giving up liquids.  A few alterations are necessary.",
     "weight": 6662,
-    "volume": 10,
+    "volume": 20,
     "price": 100,
     "to_hit": -2,
     "material": [ "flesh" ],
     "symbol": ";",
     "color": "light_green",
-    "contains": 30,
+    "contains": 40,
     "seals": true,
     "watertight": true
   },
+  {
+    "id": "gloople_tank_large",
+    "type": "CONTAINER",
+    "name": "gelatinous tank",
+    "name_plural": "gelatinous tanks",
+    "copy-from": "gloople_tank",
+    "volume": 240,
+    "contains": 240
```

Comment:
Needs to mass more than a gloople_tank - at least twice as much, since it has roughly 12x the volume and at least 4x the surface area.